### PR TITLE
July 29

### PR DESCRIPTION
**Pull request by timsehn**

### CODE REVIEW

File: emails/2022-08-05.md [hunk 1]
```diff
@@ -1,17 +1,25 @@
-
+Light summer week this week. Lots of people on vacation. Here's the update.
```
Comment (bpf120) [inline emails/2022-08-05.md:1]: Intro yourself, assume someone is reading for the first time. maybe make a joke for the people that read every week?

File: emails/2022-08-05.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Yearly, we publish a [use cases blog](https://www.dolthub.com/blog/2022-07-11-dolt-case-studies/). 
+
+[In 2020, the blog](https://www.dolthub.com/blog/2020-03-30-dolt-use-cases/) was aspirational, we didn't really have many real users. We just launched in September 2019. [In 2021](https://www.dolthub.com/blog/2021-03-09-dolt-use-cases-in-the-wild/), we were starting to see some real users and were refining the messaging around use cases. [This year](https://www.dolthub.com/blog/2022-07-11-dolt-case-studies/), we have a ton of cool stories to tell. If you have a use case, please let us know. You can reply to this email. We'll be sure to include your use case in next year's blog! 
+
+### Seattle Office
+
+[Jason](https://www.dolthub.com/team#jason) and [Zach](https://www.dolthub.com/team#zach) moved into an [office in Seattle](https://www.dolthub.com/blog/2022-08-05-dolthub-in-seattle) on Monday. I'll be making a trip up there in the next month before it starts to rain. If you're in Seattle and want to meet in person, reply to this email and I'll get you on the agenda.
+
+[![Seattle Office](..images/dolthub-seattle.png)](https://www.dolthub.com/blog/2022-08-05-dolthub-in-seattle)
```
Comment (fulghum) [inline emails/2022-08-05.md:13]:
```suggestion
[![Seattle Office](../images/dolthub-seattle.png)](https://www.dolthub.com/blog/2022-08-05-dolthub-in-seattle)
```

File: emails/2022-08-05.md (hunk 1 above)
Comment (timsehn) [reply emails/2022-08-05.md:3]: Better?

Comment (bpf120) [reply emails/2022-08-05.md:3]: love it

**Pull request merged**